src/style.css

```diff
@@ -1 +1,71 @@
 /* Styles go here */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
```

Comment:
```suggestion
.logo {
  line-height: 0;
}
```

Reply:
This size are same as in image, so no need to duplicate
You can just set this line-height to remove extra few pixels at the bottom of link